pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -1102,7 +1102,8 @@ let
 
         # enable support for device trees and overlays
         OF = option yes;
-        OF_OVERLAY = option yes;
+        # OF_OVERLAY breaks v5.10 on x86_64, see https://github.com/NixOS/nixpkgs/issues/403985
+        OF_OVERLAY = lib.mkIf ((lib.versionAtLeast version "5.15") || (!stdenv.hostPlatform.isx86_64)) (option yes);
```

Comment:
Commit message says 5.11, this says 5.15. Also, I feel like it might read better as `!(lib.versionOlder "5.15" && stdenv.hostPlatform.isx86_64)`?

Reply:
Also nixfmt fails

Reply:
Sorry, I should double-check before pushing...